File: CivitasRuby/lib/TestP2.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative "CivitasJuego"
require_relative "Dado"
require_relative "TipoSorpresa"
require_relative "TipoCasilla"
require_relative "estados_juego"
require_relative "Tablero"
require_relative "diario"
require_relative "MazoSorpresas"
require_relative "Sorpresa"

module Civitas
  class TestP2
    def self.main
      
      @jugadores = Array.new
      @jugadores << "Iván"
      @jugadores << "Mumo"
      
      @juego = CivitasJuego.new(@jugadores)
      
      @juego.actualizarInfo
      
      for i in 0..5 do
        puts @juego.mazo.siguiente.to_s
      end
      
      puts @juego.tablero.getCasilla(0).to_s
      puts @juego.tablero.getCasilla(15).to_s
      puts @juego.tablero.getCasilla(4).to_s
      puts @juego.tablero.getCasilla(17).to_s

      
      @juego.actualizarInfo
      @juego.pasarTurno
      @juego.actualizarInfo
      
      @juego.getJugadorActual.modificarSaldo(-10000)
      @juego.actualizarInfo
      
      
    end
  end
  Civitas::TestP2.main
end

File: CivitasJava/src/civitas/TestP2.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

import java.util.ArrayList;

/**
 *
 * @author ivan
 */
public class TestP2 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
       ArrayList<String> jugadores = new ArrayList<>() ;
       jugadores.add("Iván");
       jugadores.add("Mumo");
       
       CivitasJuego juego = new CivitasJuego(jugadores) ;
       
       juego.actualizarInfo();
       
       for (int i = 0 ; i < 5 ; i++){
           System.out.println(juego.getMazo().siguiente().toString()) ;
       }
       
       System.out.println(juego.getTablero().getCasilla(0).toString());
       System.out.println(juego.getTablero().getCasilla(15).toString());
       System.out.println(juego.getTablero().getCasilla(4).toString());
       System.out.println(juego.getTablero().getCasilla(17).toString());
       
       juego.actualizarInfo();
       //juego.pasarTurno() ;
       juego.actualizarInfo();
       
       System.out.println(juego.ranking()) ;
       
       juego.getJugadorActual().modificarSaldo(-10000) ;
       juego.actualizarInfo();
    }
    
}
